index.js

```diff
@@ -683,4 +683,64 @@ Peer.prototype._debug = function () {
   debug.apply(null, args)
 }
 
+// Transform constraints objects into the new format (unless Chromium)
+// This can be removed when Chromium supports the new format
```

Comment:
Can you prefix this comment with "TODO" so that it's searchable? I periodically check for "TODO" before publishing new versions and see what can be improved because of new browser releases :)